js/contact_me.js

```diff
@@ -22,6 +22,7 @@ $(function() {
       $.ajax({
         url: "https://formspree.io/mlelodwz",
         type: "POST",
+        dataType: "json",
```

Comment:
The error was not the redirect. Error was reading response data type as other than JSON.